configure_logging.sh

```diff
@@ -24,8 +26,8 @@ replace_values () {
 echo "Configuring cloudwatch logs: Starting"
 replace_values "{KURTOSIS_NAME}" "$NAME"
 #cp aws-ec2-logging-config.json /opt/aws/amazon-cloudwatch-agent/bin/config.json
-wget https://amazoncloudwatch-agent.s3.amazonaws.com/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb
-dpkg -i -E ./amazon-cloudwatch-agent.deb
+retry 10 wget https://amazoncloudwatch-agent.s3.amazonaws.com/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb
```

Comment:
Retry commands likely to fail temporarily.  We want to try multiple times because it is costly to re-run the full configure script.